sdk/security/keyvault/azkeys/models.go

```diff
@@ -18,15 +14,18 @@ type BackupKeyResult struct {
 
 // CreateKeyParameters - The key create parameters.
 type CreateKeyParameters struct {
-	// REQUIRED; The type of key to create.
+	// REQUIRED; The type of key to create. For valid values, see JsonWebKeyType.
 	Kty *KeyType
 
-	// Elliptic curve name.
+	// Elliptic curve name. For valid values, see JsonWebKeyCurveName.
 	Curve *CurveName
 
 	// The attributes of a key managed by the key vault service.
 	KeyAttributes *KeyAttributes
-	KeyOps        []*KeyOperation
+
+	// Json web key operations. For more information on possible key operations, see
```

Comment:
```suggestion
	// Json web key operations.
```

The slug is inaccurate and not realy needed - we have constants defined.

Reply:
fixed in d09d2b5